specs/Spec_for_initial_global_navigation.adoc

```diff
@@ -0,0 +1,121 @@
+# Initial UI refresh for the grafana integration
+
+Since we are having Grafana with Tendrl, the global menu has to be refined. This spec talks about the modules which we can keep from our previous build (it is available https://github.com/Tendrl/ui/tree/ui-legacy-code[ui-legacy-code] ).
+
+## Problem Description
+
+It descibes the items to be retained/added in new global menu
+  
+## Use Cases
+
+
+## Proposed change
+
+Currently the navigation has following items:-
+
+* Dashboard
+* Clusters
+* Hosts
+* Volumes
+* Pools
+* RBDs
+* Alerts
+* Admin
+
+
+The following navigation entries would be available post the refresh from the ui-legacy-code branch
```

Comment:
this whole block(26-57) is confusing. List items per comment - https://github.com/Tendrl/specifications/issues/182#issuecomment-317708538 - and state that only those items will be available in ui after refresh